scss/common/_base.scss

```diff
@@ -418,59 +418,63 @@
         opacity: .3;
     }
 
-    $loading-stroke-width: 2px !default;
-    $loading-size: $icon-size - $loading-stroke-width !default;
-    $loading-speed: 2s !default;
-    $loading-color: $accent;
-    $loading-outer-size: ($loading-size + ($loading-stroke-width * 2));
-    $loading-half-size: ($loading-outer-size / 2) !default;
-
-    .k-icon.k-i-loading {
+    // Loading indicator
+    $loading-size: 1em !default;
+    $loading-base-font-size: 16 !default;
+    $loading-base-font-size-px: 16px !default;
+    $loading-font-size: 16 !default;
+    $loading-font-size-px: 16px !default;
```

Comment:
The default loading animation size should be `$icon-size`. If developers need to change the loading animation styles, they can do so via font-size on the element (like the case in the Grid > Export PDF).
It appears that we don't really need any of these variables.